ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/util/DataStructureUtil.java

```diff
@@ -1943,14 +1943,19 @@ public SszList<SszKZGCommitment> randomSszKzgCommitmentList() {
             .orElseThrow()
             .getBlobKzgCommitmentsSchema();
     final int maxKzgCommitments =
-        spec.getGenesisSpecConfig().toVersionEip4844().orElseThrow().getMaxBlobsPerBlock();
+        spec.forMilestone(SpecMilestone.EIP4844)
+            .getConfig()
+            .toVersionEip4844()
+            .orElseThrow()
```

Comment:
I wonder if this is going to be common, we might want to put a `spec.getDenebConfig()` that returns the correct type straight from spec

Reply:
I cleaned it up a little bit but yes, I agree. We could revisit it. I raised an issue to track: https://github.com/ConsenSys/teku/issues/6787